src/main/java/Entities/Player.java

```diff
@@ -156,7 +156,8 @@ public void load(File pw) {
         }
 
     }
-    private void createPlayer(ArrayList<ArrayList> All){
```

Comment:
createPlayer was a helper function for load (which was originally in Player). We have moved load to a different class and removed createPlayer so you can also get rid of this